docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[ml-biases]]
+=== Eliminating biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. We use {ml} in science, healthcare, business, entertainment, 
+and many other fields. We make decisions based on {ml} results. It is therefore 
+crucial to keep in mind the possible biases that can corrupt results, endanger 
+fairness, and jeopardize the success of even the best-intended initiative. Some 
+of these social biases can make their way into the {ml} results more easily than 
+we might think.
+
+There are various possible biases that can negatively influence the result of a 
+{ml} analysis. *These biases typically are not present in the {ml} algorithms 
+that are used*, but rather exist in the datasets themselves or in the way the 
```

Comment:
we do have a concept of `bias` in algorithms - Bias are the simplifying assumptions made by a model to make the target function easier to learn. We might want to clarify that this doc is about the statistical/social biases and not about the bias in the algorithms.